internal/gitlab/gitlab.go

```diff
@@ -251,31 +251,26 @@ func ApplyGitlabTerraform(dryRun bool, directory string) {
 		}
 		//* AWS_SDK_LOAD_CONFIG=1
 		//* https://registry.terraform.io/providers/hashicorp/aws/2.34.0/docs#shared-credentials-file
-		os.Setenv("AWS_SDK_LOAD_CONFIG", "1")
-		os.Setenv("AWS_PROFILE", config.AwsProfile)
+		envs := map[string]string{}
+		envs["AWS_SDK_LOAD_CONFIG"]="1"
+		envs["AWS_PROFILE"]=config.AwsProfile
 		// Prepare for terraform gitlab execution
-		os.Setenv("GITLAB_TOKEN", viper.GetString("gitlab.token"))
-		os.Setenv("GITLAB_BASE_URL", viper.GetString("gitlab.local.service"))
+		envs["GITLAB_TOKEN"]=viper.GetString("gitlab.token")
+		envs["GITLAB_BASE_URL"]=viper.GetString("gitlab.local.service")
 
 		directory = fmt.Sprintf("%s/.kubefirst/gitops/terraform/gitlab", config.HomePath)
 		err := os.Chdir(directory)
 		if err != nil {
 			log.Panic("error: could not change directory to " + directory)
 		}
-		tfInitCmd := exec.Command(config.TerraformPath, "init")
-		tfInitCmd.Stdout = os.Stdout
-		tfInitCmd.Stderr = os.Stderr
-		err = tfInitCmd.Run()
-		if err != nil {
-			log.Panicf("error: terraform init for gitlab failed %s", err)
+		tfInitCmdErr := pkg.ExecShellWithVars(envs,config.TerraformPath, "init")
```

Comment:
In Go, we usually have `err` as the default variable name, it also helps to have the code more concise.